shotover-proxy/src/transforms/protect/mod.rs

```diff
@@ -68,23 +68,23 @@ fn decrypt(ciphertext: Vec<u8>, nonce: Nonce, sym_key: &Key) -> Result<MessageVa
     let decrypted_bytes =
         secretbox::open(&ciphertext, &nonce, sym_key).map_err(|_| anyhow!("couldn't open box"))?;
     //TODO make error handing better here - failure here indicates a authenticity failure
-    // let decrypted_value: Value =
-    //     serde_json::from_slice(decrypted_bytes.as_slice()).map_err(|_| anyhow!("couldn't open box"))?;
-    let decrypted_value: MessageValue =
-        bincode::deserialize(&decrypted_bytes).map_err(|_| anyhow!("couldn't open box"))?;
+    let decrypted_value: MessageValue = serde_json::from_slice(decrypted_bytes.as_slice())
+        .map_err(|_| anyhow!("couldn't open box"))?;
+    // let decrypted_value: MessageValue =
+    //     bincode::deserialize(&decrypted_bytes).map_err(|_| anyhow!("couldn't open box"))?;
```

Comment:
Is the intention all this commented out code will be replaced when the Cassandra AST is ready?

Reply:
yeah